geocoder/geocoder.cpp

```diff
@@ -156,6 +161,13 @@ void Geocoder::Context::FillResults(vector<Result> & results) const
     results.emplace_back(e.m_key /* osmId */, e.m_value /* certainty */);
   }
 
+  if (!results.empty())
+  {
+    auto const by = results.front().m_certainty;
```

Comment:
Also, it would be nice to add assertion that ```by``` is the maximal certainty.

Reply:
done